web/frontend/src/components/camera.vue

```diff
@@ -66,16 +65,14 @@ const toggleExpand = () => {
 };
 
 const renderPCD = () => {
```

Comment:
Nit: This could also use the `async/await` syntax:

```typescript
const renderPCD = async () => {
  try {
    const bytes = await new CameraClient(props.client, props.cameraName).getPointCloud();
    pointcloud = bytes;
  } catch (error) {
    toast.error(`Error getting point cloud: ${error}`);
  }
};
```